### PR TITLE
Update network stake to be pending plus rewarded stake

### LINKED ISSUE #6394: Change stake total to stake rewarded plus not rewarded
### Problem

The current `network_stake.stake_total` is the sum of all the `node_stake.stake`. Summing the stepped consensus weight is not as useful as summing the `stake_rewarded` plus `stake_not_rewarded` as [hashscan](https://hashscan.io/mainnet/nodes) does.

### Solution

* Change `NodeStakeUpdateTransactionHandler` to sum the `stake_rewarded + stake_not_rewarded`
* Add a database migration to correct historical values similarly

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
* sum pending stake and rewarded stake to calculate network total stake

**Related issue(s)**:

Fixes #6394 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/main/resources/db/migration/v1/V1.87.0__network_stake_recalculation.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+with updated_stake as (
+    select consensus_timestamp, sum(stake_rewarded + stake_not_rewarded) as stake
+    from node_stake group by consensus_timestamp
+)
+UPDATE network_stake ns
```
Comment (steven-sheehy) [inline hedera-mirror-importer/src/main/resources/db/migration/v1/V1.87.0__network_stake_recalculation.sql:5]: nit: Use lowercase like the rest

File: hedera-mirror-importer/src/main/resources/db/migration/v1/V1.87.0__network_stake_recalculation.sql
Comment (steven-sheehy) [inline hedera-mirror-importer/src/main/resources/db/migration/v1/V1.87.0__network_stake_recalculation.sql:1]: Should be V1.85.x

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-importer/src/main/resources/db/migration/v1/V1.87.0__network_stake_recalculation.sql
Comment (jnels124) [reply hedera-mirror-importer/src/main/resources/db/migration/v1/V1.87.0__network_stake_recalculation.sql:1]: I was under the impression the migration versions were modeled after the release version. I will note this for the future

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**